### PR TITLE
Added Date conversion methods

### LINKED ISSUE #7: create `toGregorianDate` & `toBikramSambat` method

### PR DESCRIPTION
**Pull request by blackkspydo**
### Description

Added date conversion methods 
-  `. toGregorian()` - BS to AD converter
- `.toBikramSambat()` - AD to BS converter

### Issue Reference

#7 

### Proposed Changes

Added date conversion methods 
-  `. toGregorian()` - BS to AD converter
- `.toBikramSambat()` - AD to BS converter

### Checklist

- [X] Lint rules are passed correctly
- [X] Code builds correctly
- [X] Created or updated tests for the new code
- [X] All existing and new tests passed
- [X] Pull request title is concise and descriptive.
- [X] Pull request is linked to the related issue.

### CODE REVIEW

File: src/BikramSambat.test.ts [hunk 1]
```diff
@@ -55,6 +55,38 @@ describe('BikramSambat Class', () => {
...
+
+  it('should properly convert to Gregorian Date', () => {
+    const sampleData = [
+      ['2075-01-01', '2018-04-14'],
+      ['2072-10-03', '2016-01-17'],
+      ['2069-11-30', '2013-03-13'],
+      ['2082-10-30', '2026-02-12'],
+      ['2086-12-30', '2030-04-13'],
+      ['2089-01-29', '2032-05-13']
+    ]
```
Comment (aabishkaryal) [inline src/BikramSambat.test.ts]: maybe we could have it in a higher scope since its used in multiple places

Comment (aabishkaryal) [reply src/BikramSambat.test.ts]: also maybe some test cases on how we handle invalid dates? like feb 30th?

File: src/BikramSambat.ts [hunk 2]
```diff
@@ -1,19 +1,22 @@
 import { format } from 'format'
 import { isDayValid, parse } from 'parser'
 import { DateFormat, InvalidDate } from 'data'
+import { getDaysFromBsNewYear } from 'utils/getDaysFromBsNewYear'
+import { addDaysToGregorianDate } from 'utils/addDaysToGregorianDate'
```
Comment (aabishkaryal) [inline src/BikramSambat.ts:5]: what do you say about having a index file at utils folder level so we don't have to worry about files? just like data folder?

**Review by aabishkaryal — CHANGES_REQUESTED** [review]

File: src/BikramSambat.ts (hunk 2 above)
Comment (blackkspydo) [reply src/BikramSambat.ts:5]: @ashiishme dada asked to export from file itself and not to use index.ts

**Review by aabishkaryal — APPROVED** [review]

**Pull request merged**